doc/user_guide/Reactive_Expressions.ipynb

```diff
@@ -515,7 +515,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "string_template = param.reactive('Hello {name}!')\n",
```

Comment:
I'm in favor of the simpler and more readable "rx()", not "param.rx()", using `from param import rx` as already being done above.